steps/mapping/ClaimMapping.step.json

```diff
@@ -120,51 +63,6 @@
         },
         "type": {
           "sourcedFrom": "\"Practitioner\""
-        },
-        "identifier": {
-          "sourcedFrom": ".",
-          "properties": {
-            "use": {
-              "sourcedFrom": "valueSetLookup(\"usual\", \"identifier-use\")/code"
-            },
-            "type": {
-              "sourcedFrom": ".",
-              "properties": {
-                "coding": {
-                  "sourcedFrom": ".",
-                  "properties": {
-                    "system": {
-                      "sourcedFrom": "\"http://terminology.hl7.org/CodeSystem/v2-0203\""
-                    },
-                    "code": {
-                      "sourcedFrom": "valueSetLookup(\"FILL\", \"identifier-type\")/code"
-                    },
-                    "display": {
-                      "sourcedFrom": "valueSetLookup(\"FILL\", \"identifier-type\")/display"
-                    },
-                    "userSelected": {
-                      "sourcedFrom": "false()"
-                    }
-                  },
-                  "targetEntityType": "#/definitions/Coding"
-                },
-                "text": {
-                  "sourcedFrom": "valueSetLookup(\"FILL\", \"identifier-type\")/display"
-                }
-              },
-              "targetEntityType": "#/definitions/CodeableConcept"
-            },
-            "system": {
-              "sourcedFrom": "\"http://hl7.org/fhir/identifier-use\""
```

Comment:
I have a feeling this system string (or the http://terminology.hl7.org/CodeSystem/v2-0203 above) is the system for this id rather than FILL, so the identifier prop should be something like "identifier__idUse".

Reply:
@damonfeldman both http://hl7.org/fhir/identifier-use\ and http://terminology.hl7.org/CodeSystem/v2-0203 are both systems for identifiers and their uses. If you take a look at http://terminology.hl7.org/CodeSystem/v2-0203 closely, you will see that it has an option called DL that means drivers license. the codes in this system are what actually define what the identifier is. According to this value set, the FILL code for "An identifier for a request where the identifier is issued by the person, or service, that produces the observations or fulfills the request." not sure that is the correct type for this claim number, but I do think it is the closes thing we have.